tests/osautoinst/test_running.pm

```diff
@@ -8,8 +8,21 @@ sub run {
     type_string "clear\n";
 }
 
+sub get_log {
```

Comment:
It's looks like something more generic than just one module function. Shouldn't we have something like this in utils.pm ? If not we need to move this one there